distribution/lib/Standard/Base/0.0.0-dev/src/Data/Range.enso

```diff
@@ -169,16 +216,27 @@ type Range
              1.up_to 6 . to_vector
     to_vector : Vector.Vector
     to_vector =
```

Comment:
not sure is better but could use `this.map i->i`

Reply:
Agree, I just wanted to avoid the microscopic overhead, but probably you are right that readability is better here and the overhead is probably none.